packages/common/src/services/integrations/jwp/JWPSubscriptionService.ts

```diff
@@ -54,12 +54,17 @@ export default class JWPSubscriptionService extends SubscriptionService {
   ): PaymentDetail => {
     const { number, exp_month, exp_year, card_name, card_type, account_id, currency } = card;
     const zeroFillExpMonth = `0${exp_month}`.slice(-2);
+
     return {
+      id: 0,
+      paymentMethodId: 0,
```

Comment:
Why you chosen this instead of making these properties optional?

Reply:
Because these types are based on Cleeng API data, when making these optional, we falsely assume these properties are not needed for the Cleeng API calls.

We can fix this properly by separating the Cleeng typings from the app typings. But this is a big change for now.